### PR TITLE
Update DG

### LINKED ISSUE #28: Update DeveloperGuide.md
Update use cases for individual features

### PR DESCRIPTION
**Pull request by hogantan**
Add feature use case

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -269,6 +269,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+**MSS**
+
+1. User enters a command to add a student with student details.
+2. Reeve saves student data into the students list and displays a success message.
+<br> Use case ends.
+
+**Extensions**
+
+* 1a. User provides input with missing compulsory fields
```
Comment (AlexCQY) [inline docs/DeveloperGuide.md]: might want to add a period here to standardize

**Review by AlexCQY — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (hogantan) [reply docs/DeveloperGuide.md]: Right! Thanks for pointing it out! I have made the necessary changes.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -122,23 +124,23 @@ Examples:
...
-* The index refers to the index number shown in the displayed person list.
+* Deletes the student at the specified `INDEX`.
+* The index refers to the index number shown in the displayed students list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `list` followed by `delete 2` deletes the 2nd student in Reeve.
+* `find Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
```
Comment (csiongn) [inline docs/UserGuide.md]: Hihi the find command syntax needs to be changed here also later.

Comment (hogantan) [reply docs/UserGuide.md]: Right! Thanks for pointing it out! I have made the necessary changes.

**Review by VaishakAnand — APPROVED** [review]
LGTM

**Review by csiongn — APPROVED** [review]
LGTM

**Review by ong-yinggao98 — APPROVED** [review]
LTGM

**Review by AlexCQY — APPROVED** [review]
LGTM

**Pull request merged**